admin/class-h5p-plugin-admin.php

```diff
@@ -162,6 +165,70 @@ function add_settings_link($links) {
     return $links;
   }
 
+  /**
+  * Retrieve a JSON representation of H5P save data for multiple users and contents, via GET request
+  *
+  */
+  public function ajax_get_users_h5p_report_data() {
+    global $wpdb;
+    $plugin = H5P_Plugin::get_instance();
+    $core = $plugin->get_h5p_instance('core');
+    $action = $_GET['action'];
+    $user_data = array();
+
+    if(!isset($action)) {
+      $core->h5pF->setErrorMessage(__('No action set', $this->plugin_slug));
+      return;
+    }
+    
+    if(strcmp($action, 'get_users_data_by_content') !== 0) {
+      $core->h5pF->setErrorMessage(__('Invalid action set', $this->plugin_slug));
```

Comment:
Is there a reason this error handling is done with this h5p library stuff compared to the catch block using http_response_code?